scripts/revcheck.php

```diff
@@ -190,6 +190,39 @@ function populateFileTreeRecurse( $lang , $path , & $output )
         }
         if ( $entry->isFile() )
         {
+            $ignoredFileNames = [
+                'README.md',      
+                'translation.xml',      
+                'readme.first',      
+                'license.xml',      
+                'extensions.xml',
+                'versions.xml',
+                'book.developer.xml',
+                'contributors.ent',
+                'contributors.xml',
+                'README',
+                'DO_NOT_TRANSLATE',
+                'rsusi.txt',
+                'missing-ids.xml',
+            ];
+            
+            $ignoredDirectories = [
+                'chmonly',
+                'internals',
+                'internals2',
```

Comment:
Didn't we remove those docs from EN?

Reply:
Yes. These should be removed from translations as well.

Reply:
Looked at the language repositories - both folders are deleted in all translations.
It seems possible to remove